src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -15,16 +15,16 @@ const Breadcrumb = React.forwardRef<HTMLOListElement, BreadcrumbProps>(
   ({ items = [], backTo, dataTestPrefixId = '' }, ref) => {
     const passDataToRouterLink = React.useCallback(
       (item: BreadcrumbItemType, index: number) => {
-        const { to, label } = item;
+        const { label, href } = item;
 
         const isLast = isLastItem(items, index);
 
-        return to && !isLast ? (
+        return !isLast ? (
           <Link
             size={2}
-            key={to}
-            href={to}
+            key={href}
```

Comment:
missing href here?

Reply:
`{...omit(item, 'label')}`

the href is included in item obj